controllers/ArticleController.js

```diff
@@ -132,10 +133,13 @@ export default class ArticleController {
   static async getAllArticles(req, res, next) {
     try {
       let articles;
-      const { tag } = req.query;
-
+      const { tag, page, limit } = req.query;
+      const pageNumber = pagination(page, limit);
       if (tag) {
         articles = await Article.findAll({
+          offset: pageNumber.offset,
+          limit: pageNumber.limit,
+          subQuery: false,
```

Comment:
What does `subQuery` do here?

Reply:
it's my understanding that when you're using limit, sequelize generates a lot of subqueries that are not needed so you set it to false to force it to make compact queries